### PR TITLE
Add language selector

### PR DESCRIPTION
**Pull request by smoylik2**
- Changed i18n init
- Added change language function
- Added change language selector
- Added script for run app from frontend folder ( yarn build:full )

### CODE REVIEW

File: frontend/src/hooks/useResources.ts [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
 
 import { ResourcesApi } from '@/api';
-import { ResourceModel } from '@/models/ResourceModel';
+import { useTranslation } from 'react-i18next';
+import { useEffect, useState } from 'react';
+import { LocalStorageService } from '@/services';
+
+type TLanguages = 'EN' | 'UA';
+
+const LANGUAGES: { [key in TLanguages]: string; } = {
```
Comment (podkolzzzin) [inline frontend/src/hooks/useResources.ts:10]: There is an API endpoint that can return the list supported of languages: is it possible to use it?

**Review by podkolzzzin — CHANGES_REQUESTED** [review]

File: frontend/src/hooks/useResources.ts (hunk 1 above)
Comment (smoylik2) [reply frontend/src/hooks/useResources.ts:10]: sure, which one path can return full list ?

Comment (podkolzzzin) [reply frontend/src/hooks/useResources.ts:10]: /api/locales should return locale keys list
Caj be checked in swagger

Comment (smoylik2) [reply frontend/src/hooks/useResources.ts]: Fixed, please check

**Review by podkolzzzin — APPROVED** [review]

**Pull request merged**